src/IntelliTect.Coalesce.CodeGeneration/IntelliTect.Coalesce.CodeGeneration.csproj

```diff
@@ -1,12 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <Description>IntelliTect.Coalesce.CodeGeneration Class Library</Description>
-    <VersionPrefix>2.0.0</VersionPrefix>
-    <Authors>intelliTect</Authors>
+    <Description>IntelliTect Coalesce code generation templates and parsers.</Description>
```

Comment:
I agree.  I tried for a bit, and couldn't get it to work-- there is zero documentation that I could find.  Perhaps this is prime blog post fodder? :+1: